io.github.appoutlet.GameOutlet.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.appoutlet.GameOutlet
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: GameOutlet
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+finish-args:
+  - --device=all
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --share=network
+  - --filesystem=xdg-desktop
+  - --socket=session-bus
```

Comment:
line 15: this is not allowed. you have to determine which dbus name you want to talk to.